static/js/tools/stat_var/explorer.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Component } from "react";
+import { StatVarSummary } from "../../shared/types";
+
+interface ExplorerPropType {
+  statVar: string;
+  displayName: string;
+  summary: StatVarSummary;
+}
+
+class Explorer extends Component<ExplorerPropType, unknown> {
+  render(): JSX.Element {
+    const placeTypeSummaryList = this.createPlaceTypeSummaryList();
+    return (
+      <div id="placeholder-container">
+        <h1 className="mb-4">{this.props.displayName}</h1>
+        <p>
+          dcid: {this.props.statVar}
+          <br></br>
+          <a href={`https://datacommons.org/browser/${this.props.statVar}`}>
+            Graph Browser node
+          </a>
+        </p>
+        This statistical variable has observations for the following places:
+        <ul>
+          {placeTypeSummaryList.map((element) => {
+            return element;
+          })}
+        </ul>
+      </div>
+    );
+  }
+
+  private createPlaceTypeSummaryList(): Array<HTMLElement> {
+    const placeTypeSummaryList = [];
+    const placeTypeSummary = this.props.summary["placeTypeSummary"];
+    for (const placeType in placeTypeSummary) {
+      const numPlaces = placeTypeSummary[placeType]["numPlaces"];
+      const subject = numPlaces > 1 ? "places" : "place";
+      const message = `${numPlaces} ${subject} of type ${placeType}`;
+      const topPlaces = placeTypeSummary[placeType]["topPlaces"];
+      placeTypeSummaryList.push(
+        <li key={placeType}>
+          {message} (e.g.{" "}
+          {topPlaces.map((element, index) => {
+            const url =
+              "https://datacommons.org/tools/timeline#" +
```

Comment:
relative path here "/tools/timeline..." should work

Reply:
done